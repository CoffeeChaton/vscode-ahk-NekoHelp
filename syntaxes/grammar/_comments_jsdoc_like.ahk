/**
* md1
; md2
* @md3
; @md4
*/
Label_exp0:
