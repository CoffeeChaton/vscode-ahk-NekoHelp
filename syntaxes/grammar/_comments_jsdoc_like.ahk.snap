>/**
#^^^ source.ahk comment.block.documentation.ahk
>* md1
#^^^^^^ source.ahk comment.block.documentation.ahk
>; md2
#^^^^^^ source.ahk comment.block.documentation.ahk
>* @md3
#^^ source.ahk comment.block.documentation.ahk comment.block.documentation.ahk
#  ^^^^ source.ahk comment.block.documentation.ahk comment.block.documentation.ahk storage.comment.block.documentation.ahk
>; @md4
#^^ source.ahk comment.block.documentation.ahk comment.block.documentation.ahk
#  ^^^^ source.ahk comment.block.documentation.ahk comment.block.documentation.ahk storage.comment.block.documentation.ahk
>*/
#^^ source.ahk comment.block.documentation.ahk
>Label_exp0:
#^^^^^^^^^^^ source.ahk labelline.ahk entity.name.function.label.ahk
>